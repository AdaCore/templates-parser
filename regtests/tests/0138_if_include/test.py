from test_support import *

gprbuild('ii')
run('ii')
