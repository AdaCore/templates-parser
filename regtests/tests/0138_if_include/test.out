
1: /content/header.tmplt - FALSE
This is header

2: header.tmplt - FALSE
This is header

3: ../content/header.tmplt - FALSE
This is header

4: ./content/header.tmplt - TRUE
This is header

DONE
